src/shared/lib/redirectHelpers.ts

```diff
@@ -107,17 +111,54 @@ export function handleCaseDO() {
     );
 }
 
+async function getCuratedNonRedundantStudyList() {
+    // Parse curated non-redundant set from AppConfig
+    let quickSelectButtons: CategorizedConfigItems = {};
+    if (AppConfig.serverConfig.skin_quick_select_buttons) {
+        try {
+            quickSelectButtons = ServerConfigHelpers.parseConfigFormat(
+                AppConfig.serverConfig.skin_quick_select_buttons
+            );
+        } catch (ex) {
+            return {};
+        }
+    }
+
+    const curatedNonRedundantStudyIdsArray = _.find(
+        quickSelectButtons,
+        (studyIds, buttonName) => {
+            const [buttonText, tooltipText] = buttonName.split('|');
+            return buttonText
+                .toLowerCase()
+                .includes('curated set of non-redundant studies');
+        }
+    );
+    if (curatedNonRedundantStudyIdsArray) {
+        // filter out studies that user doesnt have access to
+        const allStudies = await client.getAllStudiesUsingGET({});
```

Comment:
way to anticipate this.  did you manage to test this?

Reply:
yeah, there's actually one in the curated list that's not available in the public portal lol

Reply:
so, has that been failing?